src/components/productsList/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { InferGetStaticPropsType } from 'next';
+import { FilterIcon } from '../icons/FilterIcon';
+import { getStaticProps } from '@/pages/sapatos/sandalias';
+import { ProductCard } from '../productCard';
+import { useState } from 'react';
+import { Filter } from '../filter';
+
+export function ProductsList({
+  products,
+}: InferGetStaticPropsType<typeof getStaticProps>) {
+  const [open, setOpen] = useState(false);
```

Comment:
Como é uma variável que guarda um estado, é comum usarmos o is como prefixo. Ex.: `isOpen, setIsOpen`